<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopUp_InsertarID</class>
 <widget class="QDialog" name="PopUp_InsertarID">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ID NUEVAS CAMARAS</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduzca ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_id">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NÃºmero de ID restantes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_idRestantes">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_confirmar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Modelo seleccionado:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_modeloSeleccionado">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
